app/views/layouts/_top_bar.html.erb

```diff
@@ -1,57 +1,57 @@
 <div class="navigation-progress" id="navigation-progress">
 </div>
-<header class="top-bar" id="top-bar" role="banner">
-  <a href="#articles-list" class="skip-content-link">Skip to content</a>
-  <div class="top-bar__container">
-    <a href="/" class="top-bar--logo" aria-label="<%= community_name %> Home"><%= logo_svg %></a>
+<% if user_signed_in? %>
```

Comment:
We should somehow refactor this part - looks like there's a lot of the same code in both IF statements (whether user is logged in or logged out) - also the caching issue that @joshpuetz mentioned could be (not sure) related.

Reply:
💯 agree refactor because this feels really messy — I think I'll need someone who is a developer to help me out here :)